waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -29,93 +29,101 @@ type PostgresDriver* = ref object of ArchiveDriver
   futLoopPartitionFactory: Future[void]
 
 const InsertRowStmtName = "InsertRow"
-const InsertRowStmtDefinition = # TODO: get the sql queries from a file
-  """INSERT INTO messages (id, messageHash, storedAt, contentTopic, payload, pubsubTopic,
-  version, timestamp, meta) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, CASE WHEN $9 = '' THEN NULL ELSE $9 END) ON CONFLICT DO NOTHING;"""
+const InsertRowStmtDefinition =
+  """INSERT INTO messages (messageHash, pubsubTopic, contentTopic, payload,
+  version, timestamp, meta) VALUES ($1, $2, $3, $4, $5, $6, CASE WHEN $7 = '' THEN NULL ELSE $7 END) ON CONFLICT DO NOTHING;"""
+
+const SelectClause =
+  """SELECT messageHash, pubsubTopic, contentTopic, payload, version, timestamp, meta FROM messages """
 
 const SelectNoCursorAscStmtName = "SelectWithoutCursorAsc"
 const SelectNoCursorAscStmtDef =
-  """SELECT storedAt, contentTopic, payload, pubsubTopic, version, timestamp, id, messageHash, meta FROM messages
+  SelectClause &
```

Comment:
Side note: I will shortly work on an enhancement suggested by @richard-ramos where we will have only one _prepared statement_ that accepts an arbitrary number of parameters. With that, I believe we will potentially simplify that part.